Explorer/Assets/DCL/Multiplayer/Movement/Interpolation.cs

```diff
@@ -30,16 +30,17 @@ public static float Execute(float deltaTime, ref CharacterTransform transComp, r
                 transComp.Transform.position = intComp.End.position;
             }
 
-            LookAt(deltaTime, ref transComp, lookDirection, rotationSpeed, intComp.End.rotationY);
+            // Flattened to have ground plane direction only (XZ)
+            lookDirection.y = 0;
+
+            if (lookDirection != Vector3.zero)
```

Comment:
float comparisons might be unreliable, maybe we should consider using a different approach that considers a threshold?
Like
```
if(Vector3.Distance(lookDirection, Vector3.zero) <= THRESHOLD)
```

What do you think?

Reply:
I was thinking about it, but I think anything that is not equal to zero will work for the `LookAt`, even very very small floats